content/mentorship_guidelines.md

```diff
@@ -0,0 +1,145 @@
+
+# Galaxy Mentorship Network Guidelines
+
```

Comment:
Can you put the title in the metadata and also add the id, i.e. the page name?
Sth like

```
---
id: about
title: About the Galaxy Mentorshing Network Program
---
```

Reply:
Okay, I'll do that now

Reply:
I think the title might be Mentorship Guidelines since it's not the 'About page'. What do you think?

Reply:
Yes. What I pasted was the example from the about page

Reply:
Okay, on it now :)